trackpy/static.py

```diff
@@ -0,0 +1,264 @@
+from scipy.spatial import cKDTree
+import random
+import numpy as np
+from warnings import warn
+
+
+def pairCorrelation2D(feat, cutoff, fraction = 1., dr = .5, p_indices = None, ndensity=None, boundary = None,
+                            handle_edge=True):
+    """   
+    Calculate the pair correlation function in 2 dimensions.
+
+    Parameters
+    ----------
+    feat : Pandas DataFrame
+        DataFrame containing the x and y coordinates of particles
+    cutoff : float
+        Maximum distance to calculate g(r)
+    fraction : float, optional
+        The fraction of particles to calculate g(r) with. May be used to increase speed of function.
+        Particles selected at random.
+    dr : float, optional
+        The bin width
+    p_indices : sequence, optional
+        Only consider a pair of particles if one of them is in 'p_indices'.
+        Uses zero-based indexing, regardless of how 'feat' is indexed.
+    ndensity : float, optional
+        Density of particle packing. If not specified, density will be calculated assuming rectangular homogeneous
+        arrangement.
+    boundary : tuple, optional
+        Tuple specifying rectangular boundary of particles (xmin, xmax, ymin, ymax). Must be floats.
+        Default is to assume a rectangular packing. Boundaries are determined by edge particles.
+    handle_edge : boolean, optional
+        If true, compensate for reduced area around particles near the edges.
+
+    Returns
+    -------
+    r_edges : array
+        Return the bin edges
+    g_r : array
+        The values of g_r
+    """
+
+    if boundary is None:
+        xmin, xmax, ymin, ymax =  feat.x.min(), feat.x.max(), feat.y.min(), feat.y.max()
+    else:
+        xmin, xmax, ymin, ymax = boundary
+        # Disregard all particles outside the bounding box
+        feat = feat[(feat.x >= xmin) & (feat.x <= xmax) & (feat.y >= ymin) & (feat.y <= ymax)]
+
+    if ndensity is None:
+        ndensity = feat.x.count() / ((xmax - xmin) * (ymax - ymin))  #  particle packing density
```

Comment:
it should be `count - 1` because each time you drop one particle: the one you are measuring from.